modules/nlu/src/backend/module-lifecycle/on-bot-mount.ts

```diff
@@ -35,31 +38,38 @@ export function getOnBotMount(state: NLUState) {
         const entityDefs = await getCustomEntities(ghost)
         const hash = ModelService.computeModelHash(intentDefs, entityDefs)
 
-        await Promise.mapSeries(languages, async languageCode => {
-          // shorter lock and extend in training steps
-          const lock = await bp.distributed.acquireLock(makeTrainSessionKey(botId, languageCode), ms('5m'))
-          if (!lock) {
-            return
-          }
-          await ModelService.pruneModels(ghost, languageCode)
-          let model = await ModelService.getModel(ghost, hash, languageCode)
-          if (forceTrain || !model) {
-            const trainSession = makeTrainingSession(languageCode, lock)
-            state.nluByBot[botId].trainSessions[languageCode] = trainSession
+        const kvs = bp.kvs.forBot(botId)
+        await kvs.set(KVS_TRAINING_STATUS_KEY, 'training')
```

Comment:
I don't think you need to create another key, the session service already creates it

![image](https://user-images.githubusercontent.com/42552874/76106865-b8a0a600-5fa5-11ea-824e-0f92898e93a7.png)

Reply:
The keys created by the session service contain the training language (here, `en`). The session service creates one key per language.

I want to create one global key that encompasses all languages.